### PR TITLE
Kumarabd/feature/helm

### PR DESCRIPTION
**Pull request by kumarabd**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by kushthedude — COMMENTED** [review]
Is the .tgz file committed by accident? Also, can we place the chart under charts directory rather than helm?
The later is just for the sake of convention, can be ignored.

**Comment by kumarabd** [comment]
> Is the .tgz file committed by accident? Also, can we place the chart under charts directory rather than helm?
> The later is just for the sake of convention, can be ignored.

The tgz file is the helm package-d. Im having this in the repository for now to figure out which way to install the chart with the client. Im not naming it `charts` because we are having content specific to technology (helm).

**Comment by kushthedude** [comment]
The automated action need gh-page protected branch, a valid github repo admin token and a git configuration setup.
That is why I was saying to store charts in chart directory, all the automated actions follow the convention to search for charts in the chart sub-folder.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,3 +43,25 @@ jobs:
         docker push ${{ secrets.IMAGE_NAME }}:${GITHUB_REF/refs\/tags\//}
         docker tag ${{ secrets.IMAGE_NAME }}:smi ${{ secrets.IMAGE_NAME }}:smi-${GITHUB_REF/refs\/tags\//}
         docker push ${{ secrets.IMAGE_NAME }}:smi-${GITHUB_REF/refs\/tags\//}
+  helm:
+    name: Helm charts publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master') && success()
```
Comment (kushthedude) [inline .github/workflows/ci.yml:51]: @kumarabd this is not required, the following is already checked at the root.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -43,3 +43,25 @@ jobs:
...
+        uses: actions/checkout@master
```
Comment (kushthedude) [inline .github/workflows/ci.yml:52]: Let's not point the action to master who knows which commit may break it

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -43,3 +43,25 @@ jobs:
...
+        with:
+          fetch-depth: 1
+      - name: Run chart-testing (lint)
+        uses: helm/chart-testing-action@master
```
Comment (kushthedude) [inline .github/workflows/ci.yml:56]: same here

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -43,3 +43,25 @@ jobs:
...
+          command: lint
+          config: .github/workflows/ct.yaml
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.0.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (kushthedude) [inline .github/workflows/ci.yml:67]: You may also need to verify if the GITHUB_TOKEN is created or not.
/cc @leecalcote

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kumarabd) [reply .github/workflows/ci.yml:51]: @kushthedude  but this is a separate job step right? Different from `docker`

File: .github/workflows/ci.yml (hunk 2 above)
Comment (kumarabd) [reply .github/workflows/ci.yml:52]: Ultimately the code is going to merge into master. Isnt it better to have it there right away than the feature branch?

**Review by Aisuko — APPROVED** [review]
approve

**Pull request merged**